corehq/ex-submodules/soil/progress.py

```diff
@@ -9,11 +10,11 @@
 import six
 from celery.result import GroupResult
 
-from soil.exceptions import TaskFailedError
-
 TaskProgress = namedtuple('TaskProgress',
                           ['current', 'total', 'percent', 'error', 'error_message'])
 
+SOIL_ERROR_SEPARATOR = "🐜🥜🌲"
```

Comment:
![image](https://user-images.githubusercontent.com/1471773/57480669-5e906080-726e-11e9-905d-4ca5e379acf4.png)

(suggest changes as you see fit, but this one is unlikely to have conflicts and it makes HQ more "fun")

Reply:
Way to get in implicit unicode testing 🥂